### ISSUE TITLE
Update the SBMLTestSuite Runner for L3V2

### ISSUE THREAD

**Issue by matthiaskoenig** [draeger-lab/SBSCL#35]
Update the SBMLTestSuiteRunner
- [x] to use latest test files
- [x] include SBML L3V2 tests
- [x]  create parametrized tests (for every single test in the test runner to have a single test case)
- [x] refactor the test runner so collection of tests and running of a single test are separated
- [x] fail tests if they take > 60 seconds
- [ ] allow to run test suite in parallel

see `SBMLTestSuiteRunner`, `download_sbml-test-suite.sh`

**Comment by matthiaskoenig** [comment]
Also could you look into running the test-suite in parallel. This takes very long at the moment and would be great if one could run these tests in parallel via a maven flag.
The tests do not share any state and have no raise conditions so no issues should arise in parallelization. See for instance https://www.baeldung.com/maven-junit-parallel-tests

**Comment by hemilpanchiwala** [comment]
@matthiaskoenig, Should we make running SBML Test Suite tests parallel as I think this will create issues with the GLPKSolver (which we observed even before)?

**Comment by matthiaskoenig** [comment]
This is not a priority. Open a separate issue for the parallelization where you document the issues with the parallel running of the FBA tests. You can close this one here.
